activity_browser/app/bwutils/commontasks.py

```diff
@@ -94,3 +96,40 @@ def get_exchanges_data(exchanges):
         results.append(exc)
     for r in results:
         print(r)
+
+
+def get_startup_bw_dir():
+    """Returns the brightway directory as defined in the settings file.
+    If it has not been defined here, it returns the brightway default directory."""
+    if ab_settings.settings.get('custom_bw_dir') is not None:
+        return ab_settings.settings['custom_bw_dir']
+    else:  # not defined in settings
+        return get_default_bw_dir()
```

Comment:
I think that's code that i wrote, but this would be more concise and clear:
```python
return ab_settings.settings.get('custom_bw_dir', get_default_bw_dir())
```